packages/legend-application-query/style/_query-editor.scss

```diff
@@ -61,6 +61,10 @@
         padding: 0.5rem;
         margin-right: 10%;
       }
+
+      &__title:hover {
```

Comment:
usually, when one sees something like this, it means you can set `cursor: pointer` on the base CSS class